src/app/builder/episode/episode-picker.component.ts

```diff
@@ -41,8 +41,7 @@ export class EpisodePickerComponent implements OnChanges, OnInit {
 
   private getEpisodes() {
     const feedUrl = decodeURIComponent(this.feedUrl);
```

Comment:
You no longer support using `/proxy` if no environment variable is set. This is a regression.